connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -192,8 +192,8 @@ message Expression {
     }
 
     message Array {
-      DataType elementType = 1;
-      repeated Literal element = 2;
+      DataType element_type = 1;
+      repeated Literal elements = 2;
```

Comment:
When I first defined it, I confirmed to @hvanhovell  that it should be `element` rather than `elements` , so I defined it as `element`. 

https://github.com/apache/spark/pull/40218#discussion_r1125231861

At the same time, I found that we sometimes use the singular and sometimes the plural in the definition of the `repeated` field. 

https://github.com/apache/spark/blob/79b5abed8bdcd5f9657b4bcff2c5ea0c767d0bf6/connector/connect/common/src/main/protobuf/spark/connect/expressions.proto#L62-L65

https://github.com/apache/spark/blob/79b5abed8bdcd5f9657b4bcff2c5ea0c767d0bf6/connector/connect/common/src/main/protobuf/spark/connect/expressions.proto#L288

I think we should have a unified rule.